lib/neworder.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './menu.rb'
+require_relative './timestamp.rb'
+require_relative './sms.rb'
+
+class NewOrder
+
+  attr_reader :menu, :order, :sms
+
+  def initialize
+    @order = []
+    @menu = Menu.new
+    @sms = SMS.new
+  end
+
+  def add_dish(dish, number)
+    number.times { @order << { dish => @menu.items[dish] } }
```

Comment:
This looks a little tricky. I'd probably break it out like:

```ruby
number.times do
  @order << { dish => @menu.items[dish] }
end
```

Just makes it easier to scan by clarifying that the `#times` is iteration and the other braces are for a hash.